app/tasks/education_form/create_daily_spool_files.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+module EducationForm
+  class CreateDailySpoolFiles
```

Comment:
Rails and Redis/DelayedJob/ActiveJob use app/jobs for anything that is run asynchronously, so I would consider placing this there rather than app/tasks, or if this is more of a rake task to be run on a schedule, I would move this to lib/tasks